north-star-vision.md

```diff
@@ -0,0 +1,137 @@
+# North Star Vision
+
+## SIG Release Roadmap for 2021 and beyond
+
+This document contains the SIG Release Roadmap for 2021. The status tracking can
+be found at the bottom. More detailed information can be found on the [SIG
+Release][0] and [Release Engineering][1] project boards.
+
+[0]: https://github.com/orgs/kubernetes/projects/23
+[1]: https://github.com/orgs/kubernetes/projects/30
+
+### Primary Focus
+
+Establish a **consumable**, **introspectable**, and **secure** supply chain for
+Kubernetes. As a supply chain we understand the defining, building and
+publishing of Kubernetes related artifacts.
+
+1. **Consumable**: Improving the usability of artifacts by making their
+   consumption easier. This includes being process independent from external
+   companies like Google.
+1. **Introspectable**: It is clear for users at which point and how Kubernetes
+   artifacts are being built. This includes the documentation of all
+   deliverables as well as clarifying what we do not support.
+1. **Secure**: The artifacts we produce are verified for their integrity. This
+   applies to their functionality (we know what we deliver) as well as their
+   software security (we know when CVEs occur).
+
+### Deliverables
+
+The following deliverables are necessary to achieve the overall goal. Within
+the following listing, all deliverables are sorted by their priority.
+
+1. **Formalize supported release platforms (Introspectable)**
+
+   https://github.com/kubernetes/sig-release/issues/1337
+
+   Outcome: Definition of the life cycle for currently supported Kubernetes
+   artifacts and a guideline for the community about how to add new platforms.
+
+1. **Implement a Bill of Materials (BOM) for release artifacts (Introspectable /
+   Secure)**
+
+   https://github.com/kubernetes/release/issues/1837
+
+   Outcome: An automated formal verification of produced release artifacts for
+   every future release.
+
+1. **Enhance Kubernetes binary artifact management (Consumable)**
+
+   Outcome: Being able to promote files as artifacts and using this mechanism
```

Comment:
is there a linked issue or more details for what this means?

Reply:
I think this one is the epic: https://github.com/kubernetes/sig-release/issues/1372

cc @justaugustus @LappleApple if you have others in mind.